### ISSUE TITLE
CPU/RAM check are not triggering

### ISSUE THREAD

**Issue by Madeiner** [amidaware/tacticalrmm#1344]
**Server Info (please complete the following information):**
 - OS: Debian 11.5
 - Browser: Chrome
 - RMM Version (as shown in top left of web UI): 0.15.1

**Installation Method:**
  - [X] Standard
  - [ ] Docker

**Agent Info (please complete the following information):**
- Agent version (as shown in the 'Summary' tab of the agent from web UI): 2.4.0
- Agent OS: Server2019

**Describe the bug**
Cpu and ram checks are not firing even though graph shows the load is correctly detected.
![image](https://user-images.githubusercontent.com/20143460/200565757-20ca4f9c-3ee1-420a-a1d7-ba1e76217eaa.png)
I assume the check is looking at some sort of average over an unknown unit of time.

**To Reproduce**
1. Create cpu check, set warning and error values
2. Wait for cpu spike, or use a benchmark to force trigger it. For example: https://silver.urih.com/
3. Check still reports green -- all is fine.
4. Check graph. It clearly show cpu is above warning and error values
5. After a while (generally > 30m - 1 hour) the check will raise alert
7. Once alert is raised, even if cpu utiliziation returns to normal, sometimes it still reports an alert.

**Expected behavior**
Checks should fail immediately if set to fail immediately.

**Comment by wh1te909** [comment]
yes currently it gets an average from the last 15 runs and will use that average to determine if it's above the threshold. I'll tag this as enhancement to and make the 15 user customizable instead of hardcoding it, thanks!
